apps/protocol-frontend/src/hooks/useContributionCount.ts

```diff
@@ -0,0 +1,62 @@
+import { useQuery } from '@tanstack/react-query';
+import { useUser } from '../contexts/UserContext';
+import { subWeeks } from 'date-fns';
+
+const useContributionCountInRange = (args: {
+  startDate: Date | string;
+  endDate: Date | string;
+  guildIds?: number[] | null | undefined;
+  excludeUnassigned?: boolean;
+}) => {
+  const { userData, govrnProtocol: govrn } = useUser();
+
+  const { isLoading, isFetching, isError, error, data } = useQuery(
+    ['GetContributionCount', args],
```

Comment:
We want the unvalidation to be structured as ResourceVerb. So ContributionGetCount would be what we are looking for